pkgs/by-name/de/devenv/package.nix

```diff
@@ -20,12 +20,16 @@ let
   devenvNixVersion = "2.30.4";
 
   devenv_nix =
-    (nixVersions.git.overrideSource (fetchFromGitHub {
-      owner = "cachix";
-      repo = "nix";
-      rev = "devenv-${devenvNixVersion}";
-      hash = "sha256-3+GHIYGg4U9XKUN4rg473frIVNn8YD06bjwxKS1IPrU=";
-    })).overrideAttrs
+    (
+      (nixVersions.nixComponents_git.override { version = devenvNixVersion; })
```

Comment:
I guess in theory this could just use `nix_2_30`, but using `git` might still be more flexible: for example, if devenv moves to 2.33 before release.